cmd/zpool/zpool_main.c

```diff
@@ -3393,8 +3415,17 @@ print_vdev_stats(zpool_handle_t *zhp, const char *name, nvlist_t *oldnv,
 		print_iostat_histos(cb, oldnv, newnv, scale, name);
 	}
 
-	if (!(cb->cb_flags & IOS_ANYHISTO_M))
+
+	if (!(cb->cb_flags & IOS_ANYHISTO_M)) {
+		if (cb->vcdl) {
+			char *path;
+			if (nvlist_lookup_string(newnv, ZPOOL_CONFIG_PATH,
+			    &path) == 0)
+				zpool_print_cmd(cb->vcdl, zpool_get_name(zhp),
+				    path);
+		}
```

Comment:
I just changed my mind.  I will be printing the command output on the line after the histograms, since it gives the option to have the output on its own single line.  That may be helpful to some parsing scripts.